adsh/src/pages/Dummy.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import NavProfile from '../components/NavProfile'
+
+const Dummy = () => {
```

Comment:
Should we choose a better name for this file? Some name ideas, DefaultView, DefaultWrapper, etc